OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/CreateKeyEmailFragment.java

```diff
@@ -133,9 +127,10 @@ public void onClick(View v) {
                     addEmail();
                 }
             });
-        }
-        if (mAdditionalEmailModels.isEmpty() && mCreateKeyActivity.mAdditionalEmails != null) {
-            mEmailAdapter.addAll(mCreateKeyActivity.mAdditionalEmails);
+
+            if (mCreateKeyActivity.mAdditionalEmails != null) {
+                mEmailAdapter.addAll(mCreateKeyActivity.mAdditionalEmails);
+            }
```

Comment:
Why did you remove ``mAdditionalEmailModels.isEmpty()``? I think it prevented a crash so I added it recently.
Also please fix the indent here.
Maybe you should better revert these lines to the previous version.